packages/airnode-node/src/workers/local-gateways/gateways-api.ts

```diff
@@ -30,6 +30,7 @@ const httpRequestBodySchema = z.object({
 const httpSignedDataBodySchema = z.object({
   encodedParameters: z.string(),
 });
+const DEFAULT_PORT = 3000;
```

Comment:
We can opt in to default ports, because the user can decide how to bind the port when running the Airnode container. E.g.:

```
docker run --rm -v /Users/emanuel/vacuumlabs/api3/airnode/packages/airnode-examples/integrations/coingecko:/app/config --publish 8000:3000 --name airnode api3/airnode-client
```

will expose the docker internal port (3000) to port 8000, so when making request you are doing it with port 8000.